client/components/repository/common/Sidebar/Discussion/Embed.vue

```diff
@@ -54,13 +55,15 @@ export default {
   props: {
     activity: { type: Object, required: true },
     showHeading: { type: Boolean, default: false },
-    showNotifications: { type: Boolean, default: false }
+    showNotifications: { type: Boolean, default: false },
+    isVisible: { type: Boolean, default: false }
```

Comment:
This component is reused, you can't handle it like this. It needs to be handled from editor sidebar wrapper (setSeenComment based on isVisible marking)